saas/static/js/djaodjin-saas-angular.js

```diff
@@ -649,3 +674,102 @@ metricControllers.controller('metricCtrl',
             $scope.balances = data;
     });
 }]);
+
+revenueControllers.controller('revenueCtrl',
+    ['$scope', '$http', '$filter', 'urls',
+    function($scope, $http, $filter, urls) {
+
+    $scope.endOfMonth = function(date) {
+        return new Date(
+            date.getFullYear(),
+            date.getMonth() + 1,
+            0
+        )
+    };
+
+    $scope.ends_at = $scope.endOfMonth(new Date());
+
+    $scope.tabs = [
+        {key: 'amount', title: 'Amount', unit: '$'},
+        {key: 'customers', title: 'Customers'},
+    ];
+    $scope.activeTab = 'amount';
```

Comment:
Following conversation from [previous pull request](https://github.com/djaodjin/djaodjin-saas/pull/56)
$scope.tabs is initialized in a very similar way as the following code is saas/views/metrics.py:

    context.update({
            "title": "Revenue Metrics",
            "tables": [
                {"title": "Amount", "unit": "$", "key": "amount", "active": True},
                {"title": "Customers", "key": "customers"},
            ]
        })

That would indicate that the information could be passed through the Django template as an AngularJS constant. Example from coupon_list.html:

    app.constant('urls', {
        saas_api_coupon_url: "{% url 'saas_api_coupon_list' %}",
        saas_metrics_coupons_url: "{% url 'saas_metrics_coupons' organization %}",
    });